src/paddle.js

```diff
@@ -0,0 +1,19 @@
+(function(exports) {
+
+function Paddle(canvas) {
+  this.canvas = canvas;
+  this.paddleHeight = 60;
+  this.paddleWidth = 10;
+}
```

Comment:
dear esteemed colleagues, please may you kindly amend as per below at your convenience:

function Ball(canvas) {
  this.canvas = canvas
  this.ctx = this.canvas.getContext("2d");
}

thank you very much.

kind regards,

Alistair :)